.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,25 @@
+name: "Semgrep App"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - staging
```

Comment:
@thypon Do you mind adding your feature branch to this just temporarily so I can get a sense of what this does exactly?  Clearly it's going to spawn some sort of audit, I just want to get a sense of how it impacts workflows etc.